sklearn/svm/_classes.py

```diff
@@ -918,7 +920,9 @@ class NuSVC(BaseSVC):
         default='rbf'
          Specifies the kernel type to be used in the algorithm.
          If none is given, 'rbf' will be used. If a callable is given it is
-         used to precompute the kernel matrix.
+         used to precompute the kernel matrix. For an intuitive
+        visualization of different kernel types see
```

Comment:
The CI is failing because of the indentation here, which was not your fault. Can you please remove one blank space from current lines 921 to 923?